introduction.md

```diff
@@ -0,0 +1 @@
+Hello i'm @elchesco and this is a fork for the course, :)
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.